app/models/concerns/hyrax/with_events.rb

```diff
@@ -1,11 +1,16 @@
 module Hyrax
+  # A mixin module intended to provide an interface into creating the paper trail
+  # of activity on the target model of the mixin.
+  #
+  # @see Hyrax::Event
   module WithEvents
```

Comment:
I was looking at the structure of this module. I find it peculiar that the methods of this module call a private method `event_stream` which in turn calls a public method. Can we "privatize" `stream`? I think I'll create a PR on this branch to test that very thing.